app/src/main/java/sopt/org/millie/presentation/bookdetail/component/book/BookDetailReview.kt

```diff
@@ -0,0 +1,103 @@
+package sopt.org.millie.presentation.bookdetail.component.book
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.width
+import androidx.compose.material3.Icon
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.res.vectorResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import sopt.org.millie.R
+import sopt.org.millie.core.designsystem.theme.MillieTheme
+import sopt.org.millie.presentation.bookdetail.component.tag.BookDetailLikedTag
+import sopt.org.millie.presentation.bookdetail.model.BookReviewModel
+
+@Composable
+fun BookDetailReview(
+    bookReview: BookReviewModel,
+    onLikeClick: () -> Unit,
+    modifier: Modifier = Modifier,
+) {
+    Column(
+        modifier = modifier.fillMaxWidth(),
+        verticalArrangement = Arrangement.spacedBy(10.dp),
+    ) {
+        UserInfo(
+            username = bookReview.username,
+        )
+
+        Text(
+            text = "${bookReview.dateOfReview}",
```

Comment:
```suggestion
            text = bookReview.dateOfReview,
```
요거 어차피 String 값이라 이렇게 작성해도 좋을 것 같아요 !!